extra/python/rx.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+# Description: Simple netmap rx example
+# Author: Faisal T. Abu-Nimeh (abunimeh@)
+
+import netmap
+import select
+
+IF_NAME = 'em0'
+
+d=netmap.NetmapDesc('netmap:' + IF_NAME)
+rxr = d.receive_rings[0] # single ring example
+# use poll
+poller = select.poll()
+poller.register(d.getfd(), select.POLLIN)
+
+while True:
+    ready_list = poller.poll() # wait for data
+    if len(ready_list) == 0:
+        print("Timeout occurred")
+        break;
+
+    if not rxr.empty(): # only process nonempty ring
+        i = rxr.cur
+
+        flen = rxr.slots[i].len
+        if flen > rxr.nr_buf_size: continue # drop large packets
+        fbuf = rxr.slots[i].buf[:flen] # this will fail if flen > nr_buf_size, i.e., nr_buf_size must be >= mtu
```

Comment:
It can.

This actually occured to me that's why I committed this example.
I am running FreeBSD 10.3-RELEASE 
with the default sysctl dev.netmap.buf_size 
my ifconfig mtu was 9000 and my ethernet frame payload is around 8944.
flen was equal to the payload, i.e., 8944
len(fbuf) was equal to dev.netmap.buf_size (thas around 2048)
There were no flags indicating that I have fragments and the remaining values were in rxr.slots[i+1], rxr.slots[i+2], etc.